components/terraform/cognito/main.tf

```diff
@@ -0,0 +1,136 @@
+resource "aws_cognito_user_pool" "cognito_user_pool" {
+    name = var.cognito_user_pool_name
+
+    auto_verified_attributes = [
+        "email",
+    ]
+
+    email_configuration {
+        source_arn             = var.ses_sysdev_arn
+        reply_to_email_address = var.ses_sysdev_email
+        email_sending_account  = "DEVELOPER"
+    }
+
+    lambda_config {
+        pre_sign_up = "arn:aws:lambda:eu-west-2:${var.account_id}:function:cognito-signup-validation"
+    }
+
+    password_policy {
+        minimum_length    = 8
```

Comment:
Would it be a bad thing to increase the max length to 12?

Reply:
ETNA users are old people you think it's required, I will leave it to business team